packages/grafana-ui/src/components/Select/getSelectStyles.ts

```diff
@@ -43,6 +43,7 @@ export const getSelectStyles = stylesFactory((theme: GrafanaTheme) => {
       font-size: ${theme.typography.size.sm};
       color: ${theme.colors.textWeak};
       white-space: normal;
+      line-height: ${theme.typography.lineHeight.md};
```

Comment:
is this not the default body line height?

Reply:
Yep, that's correct. I guess it's more of a reset. My reasoning for adding it was due to `.filter-table` class used on the permissions list declares [a line-height](https://github.com/grafana/grafana/blob/master/public/sass/components/_filter-table.scss#L33). I considered reworking the table styles but thought it better if the dropdown descriptions looked consistent wherever they're used.

I spoke with @jessover9000 before adding it and we came to the conclusion that of the lineHeight theme values we have right now this one looks the best.

Reply:
ok, yea that filter table line-height is probably not the best solution, but I see why this is needed now